src/test/java/org/openstreetmap/atlas/geography/sharding/SlippyTileTest.java

```diff
@@ -66,9 +68,9 @@ public void testMultiPolygonShards()
                 .wkt("MULTIPOLYGON (((40 40, 20 45, 45 30, 40 40)),"
                         + "((20 35, 10 30, 10 10, 30 5, 45 20, 20 35),"
                         + "(30 20, 20 15, 20 25, 30 20)))");
-        @SuppressWarnings("unchecked")
-        final Set<Shard> coveredShards = (Set<Shard>) new SlippyTileSharding(8)
-                .shards(multiPolygon);
+        final Set<Shard> coveredShards = StreamSupport
+                .stream(new SlippyTileSharding(8).shards(multiPolygon).spliterator(), false)
```

Comment:
Did you notice a performance increase using `spliterator()`?

Reply:
Not sure. This was just implement set construction in one line. The previous test used the knowledge that the returned iterable was a Set and directly cast to that. That broke when I got rid of the default.